lib/extralite.rb

```diff
@@ -41,5 +41,34 @@ def pragma_set(values)
     def pragma_get(key)
       query("pragma #{key}")
     end
+
+    module Status
+      LOOKASIDE_USED      =  0
+      CACHE_USED          =  1
+      SCHEMA_USED         =  2
+      STMT_USED           =  3
+      LOOKASIDE_HIT       =  4
+      LOOKASIDE_MISS_SIZE =  5
+      LOOKASIDE_MISS_FULL =  6
+      CACHE_HIT           =  7
+      CACHE_MISS          =  8
+      CACHE_WRITE         =  9
+      DEFERRED_FKS        = 10
+      CACHE_USED_SHARED   = 11
+      CACHE_SPILL         = 12
+    end
+  end
+
+  module Status
+    MEMORY_USED          = 0
+    PAGECACHE_USED       = 1
+    PAGECACHE_OVERFLOW   = 2
+    SCRATCH_USED         = 3  # NOT USED
+    SCRATCH_OVERFLOW     = 4  # NOT USED
+    MALLOC_SIZE          = 5
+    PARSER_STACK         = 6
+    PAGECACHE_SIZE       = 7
+    SCRATCH_SIZE         = 8  # NOT USED
+    MALLOC_COUNT         = 9
```

Comment:
I'd rather preserve the full constant names as in the sqlite3 C API, e.g.: `Extralite::SQLITE_STATUS_MEMORY_USED` and `Extralite::SQLITE_DBSTATUS_LOOKASIDE_USED`.